### PR TITLE
feat: Add Lock Viewer Cursors Setting UI

### PR DESCRIPTION
**Pull request by KDSBrowne**
### What does this PR do?
Adds the UI portion for the locking of viewers cursor on the whiteboard.
When locked viewers will only be able to see cursors for the presenter and themself.
Also removes the unlocked label from the feature toggles in the lock viewers modal. 
image

### Closes Issue(s)
#14636

### CODE REVIEW

File: bigbluebutton-html5/public/locales/en.json [hunk 1]
```diff
@@ -696,6 +696,7 @@
     "app.lock-viewers.button.cancel": "Cancel",
     "app.lock-viewers.locked": "Locked",
     "app.lock-viewers.unlocked": "Unlocked",
```
Comment (antobinary) [inline bigbluebutton-html5/public/locales/en.json:698]: Is this used anywhere else?

Comment (KDSBrowne) [reply bigbluebutton-html5/public/locales/en.json]: It's been removed in the latest commit. Wasn't being used anywhere else.

**Comment by ramonlsouza** [comment]
Had a blue screen after changing "See other viewers cursors" lock value + pressing apply in the lock modal

cursor-lock-blue-screen

**Comment by antobinary** [comment]
@ramonlsouza did you use en.json from source or from packages? bbb-html5.nginx defaults on picking en.json from /usr/share/meteor/bundle/...  and afaik we still need to manually point to the local one (so Meteor serves it in dev env)

**Comment by ramonlsouza** [comment]
> @ramonlsouza did you use en.json from source or from packages? bbb-html5.nginx defaults on picking en.json from /usr/share/meteor/bundle/... and afaik we still need to manually point to the local one (so Meteor serves it in dev env)

from source

**Pull request merged**